src/Cafe/CMakeLists.txt

```diff
@@ -9,21 +8,41 @@ set_property(TARGET CemuCafe PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CON
 
 target_precompile_headers(CemuCafe PRIVATE ../Common/precompiled.h)
 
-target_include_directories(CemuCafe PRIVATE ../)
+target_include_directories(CemuCafe PUBLIC "../")
 
 #target_link_libraries(CemuCafe ZArchivexx)
 #target_link_libraries(CemuCafe CemuCommon CemuCore CemuConfig CemuUtil CemuResource)
 #target_link_libraries(CemuCafe OpenSSL::SSL)
 #target_link_libraries(CemuCafe ZLIB::ZLIB)
-#target_link_libraries(CemuCafe imgui::imgui)
 #target_link_libraries(CemuCafe imguiImpl)
 #target_link_libraries(CemuCafe pugixml pugixml::static pugixml::pugixml)
 #target_link_libraries(CemuCafe libzip::zip)
-target_link_libraries(CemuCafe glslang SPIRV)
-target_link_libraries(CemuCafe ih264d zarchive)
 #target_link_libraries(CemuCafe zstd::libzstd_static)
 
+target_link_libraries(CemuCafe PRIVATE
+	CemuAsm
+	CemuAudio
+	CemuCommon
+	CemuComponents
+	CemuConfig
+	CemuGui
+	CemuInput
+	CemuResource
+	CemuUtil
+	imguiImpl
+	imgui::imgui
+	PNG::PNG
+	glslang::SPIRV
```

Comment:
Had to change this back to `SPIRV` on Gentoo.

Reply:
Probably because Gentoo ships an outdated glslang package, like vcpkg